source/braille.py

```diff
@@ -792,7 +792,7 @@ def getControlFieldBraille(info, field, ancestors, reportStart, formatConfig):
 		return (_("%s end") %
 			getBrailleTextForProperties(role=role))
 
-def getFormatFieldBraille(field, isAtStart, formatConfig):
+def getFormatFieldBraille(field, isAtStart, formatConfig, fieldCache=None):

```

Comment:
Any suggestions for the `isAtStart` parameter doc? You have been working with that recently I think.